resources/inspec.rb

```diff
@@ -1,34 +0,0 @@
-# encoding: utf-8
-
-provides :inspec
-resource_name :inspec
-
-property :version, String, default: 'latest'
-
-default_action :install
-
-# installs inspec
-action :install do
-  converge_by 'install/update inspec' do
-    chef_gem 'inspec' do
-      version new_resource.version if new_resource.version != 'latest'
-      compile_time true
-      action :install
-    end
-  end
-
-  converge_by 'verifies the inspec version' do
-    verify_inspec_version version
-  end
-end
-
-def verify_inspec_version(inspec_version)
-  require 'inspec'
-  # check that we have the right inspec version
-  if Inspec::VERSION != inspec_version && inspec_version !='latest'
```

Comment:
@jeremymv2 Do you think we should keep that warning. May be very helpful for debugging?